python/cugraph/tests/test_betweenness_centrality.py

```diff
@@ -258,64 +248,56 @@ def prepare_test():
 # =============================================================================
 @pytest.mark.parametrize('graph_file', TINY_DATASETS)
```

Comment:
get datasets from utils

Reply:
This was a case where the test defined a different dataset than others.  The next step is to start addressing issues what the common test set does not work across all test cases